components/collections/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_collections"
+description = "Collection of API for use in ICU libraries."
+version = "0.6.0"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+readme = "README.md"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "LICENSE"
+categories = ["internationalization"]
+# Keep this in sync with other crates unless there are exceptions
+include = [
+    "src/**/*",
+    "examples/**/*",
+    "benches/**/*",
+    "tests/**/*",
+    "Cargo.toml",
+    "LICENSE",
+    "README.md"
+]
+
+[package.metadata.cargo-all-features]
+skip_optional_dependencies = true
+# Bench feature gets tested separately and is only relevant for CI
+denylist = ["bench"]
+
+[package.metadata.docs.rs]
+all-features = true
+
+[dependencies]
+icu_codepointtrie = { path = "codepointtrie", features = ["serde"] }
```

Comment:
So, I think we want to actually move `CodePointTrie` into the `icu_collections` crate (and the other two). One of the reasons was that we wanted to enable better code sharing between the collections, and also because we didn't want to have new crates every time we add a new data structure.

The way to do this is to just make `codepointtrie` a module of `icu_collections` (in the lib.rs file) instead of a re-export, and delete the Cargo.toml and LICENSE files inside `collections/codepointtrie`.

We want to keep `collections/codepointtrie/builder` a standalone crate because of its dependency on wasmer.

Reply:
Will the `CodePointTrie` source code also be removed from its current location, and live in the flat `icu_collections` directory? Or is this a difference in lib.rs semantics?

Reply:
`components/collections/codepointtrie/src/{lib.rs, **/*.rs}` -> 
`components/collections/src/codepointtrie/{mod.rs, **/*.rs}`